packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -148,35 +148,30 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
     if (isDisabled !== undefined) {
       disabled = isDisabled;
     }
-    const describedBy = id + '-option';
+    const optionId = id + '-option';
+    const labelId = id + '-label';
 
     return (
       <div className={css('ms-Dropdown-container')}>
         {label && (
-          <Label className={css('ms-Dropdown-label')} id={id + '-label'} htmlFor={id} required={required}>
+          <Label className={css('ms-Dropdown-label')} id={labelId} htmlFor={id} required={required}>
             {label}
           </Label>
         )}
         <KeytipData keytipProps={keytipProps} disabled={disabled}>
           {(keytipAttributes: any): JSX.Element => (
-            <div
+            <button
               {...keytipAttributes}
               data-is-focusable={!disabled}
               ref={this._dropDown}
               id={id}
               tabIndex={disabled ? -1 : 0}
               aria-expanded={isOpen ? 'true' : 'false'}
-              role="listbox"
-              aria-live={disabled || isOpen ? 'off' : 'assertive'}
               aria-label={ariaLabel}
-              aria-describedby={mergeAriaAttributeValues(describedBy, keytipAttributes['aria-describedby'])}
-              aria-activedescendant={
```

Comment:
This change here regresses #4432

Reply:
I would suspect the removal aria-live item moreso. While aria-activedescendant seems it might work as a workaround, it is invalid, as aria-activedescendant is not allowed on the button that opens the actual listbox. The w3 example just opens the dropdown if you hit up/down, selecting the item moved to, removing the issue by design. This seems to be a fundamental issue of the dropdown, should I leave the aria-live workaround for now, and it can be removed if the dropdown is fixed to conform to standard?

Reply:
The w3 implementation of the dropdown implements a different type of dropdown. Unfortunately, removing the ability to press up/down without opening the popup is probably going to be considered a regression by many downstream apps. 

I am investigating and taking a look to see if there's a middle ground here.